src/main/java/io/jenkins/plugins/luxair/ImageTag.java

```diff
@@ -48,9 +48,9 @@
         if (m.find()) {
             rtn[0] = m.group(1);
             rtn[1] = m.group(2);
-            logger.info("realm:" + rtn[0] + ": service:" + rtn[1] + ":");
+            logger.info(() -> "realm:" + rtn[0] + ": service:" + rtn[1] + ":");
```

Comment:
Why using lambda here ? Keep it simple

Reply:
Sorry, this was SonarLint messing with me. I'll fix this and its repetitions